allennlp/data/token_indexers/token_indexer.py

```diff
@@ -74,8 +74,8 @@ def pad_token_sequence(self,
         """
         raise NotImplementedError
 
-    @staticmethod
-    def from_params(params: Params):
+    @classmethod
```

Comment:
good question. in the `TokenCharactersIndexer` subclass, `from_params` does use `cls`.

so mypy gets angry if the base class doesn't as well.